kura/org.eclipse.kura.container.provider/OSGI-INF/metatype/org.eclipse.kura.container.provider.ContainerInstance.xml

```diff
@@ -123,6 +123,15 @@
             type="String" cardinality="1"
             required="false" default="" />
             
+        <AD id="container.runtime"
+            name="Runtime"
+            description="Specify the runtime the container can use."
```

Comment:
I would add a little bit more details here... something like:

```suggestion
            description="Specifies the fully qualified name of an alternate OCI-compatible runtime, which is used to run commands specified by the 'run' instruction. Example: 'nvidia' corresponds to '--runtime=nvidia'."
```

Reply:
Also: does this work with Podman? Docker expects fully qualified names, Podman seems[ to only accept paths](https://docs.podman.io/en/latest/markdown/podman-farm-build.1.html#runtime-path)

Reply:
Nevermind, I was looking at the wrong docs: https://docs.podman.io/en/latest/markdown/podman.1.html#runtime-value